requirements.txt

```diff
@@ -0,0 +1,86 @@
+attrs==19.3.0
```

Comment:
In a future PR, you should replace this with the actual software requirements, not the full pip freeze list.